packages/connectivity/connectivity_macos/example/android/app/build.gradle

```diff
@@ -0,0 +1,58 @@
+def localProperties = new Properties()
```

Comment:
Why do we need the _macos example building for Android?

Reply:
https://github.com/flutter/flutter/issues/39657 :(